eskipfile/watch_test.go

```diff
@@ -45,18 +44,12 @@ func deleteFile(t *testing.T) {
 }
 
 func createFileWith(content string, t *testing.T) {
-	tmpfile, err := ioutil.TempFile("", "watch_test-*")
```

Comment:
Should we do this changes in separate PR? Would be easier to dig git history down the line

Reply:
Ok, I'll split it out.

Reply:
https://github.com/zalando/skipper/pull/1799 Split it out to here